extensions/resource-deployment/src/interfaces.ts

```diff
@@ -62,12 +62,15 @@ export interface DialogFieldInfo {
 	required: boolean;
 	options: string[];
 	placeHolder: string;
+	userName?: string;
+	userNameVariableName?: string;
```

Comment:
i’m a little confused about the naming here. any chance you could add a comment?

Reply:
I removed the usernameVariableName since we don't need that anymore and added comment for user name.